templates/azure/teamsFx/webapp.bicep

```diff
@@ -7,19 +7,16 @@ param provisionOutputs object
 param currentAppSettings object
 
 var botWebAppName = split(provisionOutputs.webAppOutput.value.resourceId, '/')[8]
-
-
 var botAadAppClientId = provisionParameters['botAadAppClientId']
-
 var botAadAppClientSecret = provisionParameters['botAadAppClientSecret']
 
-var botId = provisionParameters['botAadAppClientId']
-
 resource botWebAppSettings 'Microsoft.Web/sites/config@2021-02-01' = {
   name: '${botWebAppName}/appsettings'
   properties: union({
-    BOT_ID: botAadAppClientId // ID of your bot
-    BOT_PASSWORD: botAadAppClientSecret // Secret of your bot
-    IDENTITY_ID: provisionOutputs.identityOutput.value.identityClientId // User assigned identity id, the identity is used to access other Azure resources
-  }, currentAppSettings)
+      MicrosoftAppId: botAadAppClientId // ID of your bot
+      MicrosoftAppPassword: botAadAppClientSecret // Secret of your bot
+      MicrosoftAppType: 'MultiTenant'
+      ConnectionName: 'azdo2'
+      IDENTITY_ID: provisionOutputs.identityOutput.value.identityClientId // User assigned identity id, the identity is used to access other Azure resources
+    }, currentAppSettings)
```

Comment:
I have a question on "As we adding more feature to this Teams App, do we need to add some unit test cases"?

Reply:
Yes. I think we need to setup tests for the app. And I think this should be a sample for setting up unit test for Teams App.